src/components/People/PeopleList.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import { Person } from '../../types';
+import { PeopleTable } from './PeopleTable';
+import { SearchLink } from '../SearchLink';
+
+type Props = {
+  people: Person[];
+  personId: string;
+  getFather: (name: string | null) => Person | undefined;
+  getMother: (name: string | null) => Person | undefined;
+};
+const COLUMN_HEADERS = ['Name', 'Sex', 'Born', 'Died', 'Mother', 'Father'];
+
+type SortableFields = 'name' | 'sex' | 'born' | 'died';
+
+export const PeopleList: React.FC<Props> = ({
+  people,
+  personId,
+  getFather,
+  getMother,
+}) => {
+  const [sortField, setSortField] = useState<SortableFields | null>(null);
+  const [sortOrder, setSortOrder] = useState<'asc' | 'desc' | null>(null);
```

Comment:
why do we need a state for these fields, you can just get them from the query params